common-ui/src/main/res/values/design-system-theming.xml

```diff
@@ -15,40 +14,52 @@
+        <item name="textAppearanceHeadline2">@style/TextAppearance.DuckDuckGo.Headline2</item>
+        <item name="textAppearanceHeadline3">@style/TextAppearance.DuckDuckGo.Headline3</item>
+        <item name="textAppearanceHeadline4">@style/TextAppearance.DuckDuckGo.Headline4</item>
+        <item name="textAppearanceHeadline5">@style/TextAppearance.DuckDuckGo.Headline5</item>
+        <item name="textAppearanceBody1">@style/TextAppearance.DuckDuckGo.Body1</item>
+        <item name="textAppearanceBody2">@style/TextAppearance.DuckDuckGo.Body2</item>
+        <item name="textAppearanceButton">@style/TextAppearance.DuckDuckGo.Button1</item>
+        <item name="textAppearanceButton1">@style/TextAppearance.DuckDuckGo.Button1</item>
+        <item name="textAppearanceButton2">@style/TextAppearance.DuckDuckGo.Button2</item>
+        <item name="textAppearanceCaption">@style/TextAppearance.DuckDuckGo.Caption</item>
 
-        <item name="android:windowBackground">?attr/colorPrimaryDark</item>
-        <item name="android:windowActivityTransitions">true</item>
-        <item name="android:statusBarColor">?toolbarBgColor</item>
-        <item name="windowActionBar">false</item>
-        <item name="windowNoTitle">true</item>
-        <item name="android:colorEdgeEffect">@color/newBlack</item>
+        <item name="textAppearanceTitle">@style/Typography.DuckDuckGo.Title</item>
+        <item name="textAppearanceH1">@style/Typography.DuckDuckGo.H1</item>
+        <item name="textAppearanceH2">@style/Typography.DuckDuckGo.H2</item>
+        <item name="textAppearanceH3">@style/Typography.DuckDuckGo.H3</item>
+        <item name="textAppearanceH4">@style/Typography.DuckDuckGo.H4</item>
+        <item name="textAppearanceH5">@style/Typography.DuckDuckGo.H5</item>
 
-        <!-- Old System Components (will be refactored soon) -->
+        <!-- Android Components -->
         <item name="actionOverflowMenuStyle">@style/PopUpOverflowMenu</item>
         <item name="actionOverflowButtonStyle">@style/PopUpOverflowButton</item>
-        <item name="snackbarButtonStyle">@style/snackbarButtonStyle</item>
-        <item name="alertDialogTheme">@style/AlertDialogTheme</item>
-        <item name="android:alertDialogTheme">@style/AlertDialogTheme</item>
+        <item name="bottomSheetDialogTheme">@style/Widget.DuckDuckGo.BottomSheetDialog</item>
+        <item name="tabStyle">@style/Widget.DuckDuckGo.TabLayout</item>
         <item name="radioButtonStyle">@style/Widget.DuckDuckGo.RadioButton</item>
-        <item name="autoCompleteTextViewStyle">@style/AutoCompleteTextViewStyle</item>
         <item name="sliderStyle">@style/Widget.DuckDuckGo.Slider</item>
         <item name="switchStyle">@style/Widget.DuckDuckGo.Switch</item>
-
-        <!-- Design System Themes -->
-        <item name="bottomSheetDialogTheme">@style/Widget.DuckDuckGo.BottomSheetDialog</item>
+        <item name="snackbarButtonStyle">@style/Widget.DuckDuckGo.Snackbar</item>
```

Comment:
I tested snackbar style in dark mode and it's not correct. Until we set all new design system theme colors correctly we should probably set up a Snackbar style. Just now, we have `Widget.DuckDuckGo.Snackbar` which is the style for the action button but I think we should rename it to `Widget.DuckDuckGo.SnackbarButton` and create `<style name="Widget.DuckDuckGo.Snackbar" parent="Widget.MaterialComponents.Snackbar">` with the correct snackbar colors for both modes (light and dark)